ios/Tealeaf/TLFResources.bundle/TealeafBasicConfig.plist

```diff
@@ -13,15 +13,15 @@
 	<key>JavaScriptInjectionDelay</key>
 	<integer>100</integer>
 	<key>KillSwitchEnabled</key>
-	<false/>
+	<true/>
 	<key>KillSwitchMaxNumberOfTries</key>
 	<integer>3</integer>
 	<key>KillSwitchTimeInterval</key>
 	<integer>180</integer>
 	<key>KillSwitchTimeout</key>
 	<integer>5</integer>
 	<key>KillSwitchUrl</key>
-	<string></string>
+	<string>https://lib-us-2.brilliantcollector.com/collector/switch/b6c3709b7a4c479bb4b5a9fb8fec324c</string>
```

Comment:
we don't appear to have traditionally shipped this, is this something we should be including?

Reply:
If killswitch is enabled, then will need to add a base killswitch url.